ComputeDataDrivenCrossSection.py

```diff
@@ -27,13 +27,28 @@
 parser.add_argument("--FD", action='store_true', help='flag to compute FD cross section', default=False)
 parser.add_argument("--Dplus", action='store_true', help='flag to compute D+ cross section', default=False)
 parser.add_argument("--Ds", action='store_true', help='flag to compute Ds cross section', default=False)
+parser.add_argument("--Lc", action='store_true', help='flag to compute Lc cross section', default=False)
 parser.add_argument("--batch", action='store_true', help='suppress video output', default=False)
+parser.add_argument("--corr", action='store_true', help='assume raw yield and (prompt/feed-down) fraction uncertanties as correlated', default=False)
+parser.add_argument("--uncorr", action='store_true', help='assume raw yield and (prompt/feed-down) fraction uncertanties as uncorrelated', default=False)
+parser.add_argument("--anticorr", action='store_true', help='assume raw yield and (prompt/feed-down) fraction uncertanties as anti correlated', default=False)
 args = parser.parse_args()
 
 # Define arguments (systematic uncertainties from AliHFSystErr)
 systErr = AliHFSystErr()
 systErr.SetIsDataDrivenFDAnalysis(True)
 
+RYFractionCorrelation = 'corr'
+if sum([args.corr, args.uncorr, args.anticorr]) == 0:
+    print("WARNING: Raw Yield and feed-down uncertanties correlation is not specified. Options are --corr --uncorr --anticorr. Default is --corr.")
```

Comment:
do not exceed the limit of 120 characters in https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/.pylintrc